src/database/any.rs

```diff
@@ -300,19 +312,26 @@ impl BatchDatabase for AnyDatabase {
             AnyDatabase::Memory(inner) => inner.begin_batch().into(),
             #[cfg(feature = "key-value-db")]
             AnyDatabase::Sled(inner) => inner.begin_batch().into(),
+            #[cfg(feature = "sqlite")]
+            AnyDatabase::Sqlite(inner) => inner.begin_batch().into(),
         }
     }
     fn commit_batch(&mut self, batch: Self::Batch) -> Result<(), Error> {
         match self {
             AnyDatabase::Memory(db) => match batch {
                 AnyBatch::Memory(batch) => db.commit_batch(batch),
                 #[cfg(feature = "key-value-db")]
```

Comment:
Also need to add the `sqlite` feature here for when `--no-default-features` is used:
```suggestion
                #[cfg(any(feature = "key-value-db", feature = "sqlite"))]
```